js/components/app_details.tag

```diff
@@ -37,5 +33,27 @@
             $('#right-block').addClass('animated slideInRight');
         });
     </script>
-    
-</app-details>
\ No newline at end of file
+
+</app-details>
+
+<member>
+    <div class="member-block col-md-4 no-side-padding text-center">
+        <img class="member-avatar" src="{thumbUrl}" />
+        <p class="member-name">{name}</p>
+    </div>
+
+    <script>
+        this.on('mount', function() {
+            var self = this;
+//            var url = './chau.json';
+            var url = '/airtable/v0/appXISBe0Du86nEiX/People/' + this.id;
```

Comment:
Correct. It definitely does not work locally, which is why I would just copy/paste the contents into a JSON file locally for debugging. Yeah, pretty ghetto.